yt_dlp/extractor/manyvids.py

```diff
@@ -1,113 +1,100 @@
+                'thumbnail': 'https://cdn5.manyvids.com/php_uploads/video_images/DestinyDiaz/thumbs/thumb_Hs26ATOO7fcZaI9sx3XT_screenshot_001.jpg',
+                'uploader': 'DestinyDiaz',
+                'view_count': int,
+                'like_count': int,
+                'release_timestamp': 1508419904,
+                'tags': ['AdultSchool', 'BBW', 'SFW', 'TeacherFetish'],
+                'release_date': '20171019',
+                'duration': 3167.0,
+            },
         },
-    }, {
-        # full video
-        'url': 'https://www.manyvids.com/Video/935718/MY-FACE-REVEAL/',
-        'md5': 'bb47bab0e0802c2a60c24ef079dfe60f',
-        'info_dict': {
-            'id': '935718',
-            'ext': 'mp4',
-            'title': 'MY FACE REVEAL',
-            'description': 'md5:ec5901d41808b3746fed90face161612',
-            'uploader': 'Sarah Calanthe',
-            'view_count': int,
-            'like_count': int,
+        {
+            # full video
+            'url': 'https://www.manyvids.com/Video/935718/MY-FACE-REVEAL/',
+            'md5': 'bb47bab0e0802c2a60c24ef079dfe60f',
+            'info_dict': {
+                'id': '935718',
+                'ext': 'mp4',
+                'title': 'MY FACE REVEAL',
+                'description': 'md5:ec5901d41808b3746fed90face161612',
+                'thumbnail': 'https://ods.manyvids.com/1001061960/3aa5397f2a723ec4597e344df66ab845/screenshots/thumbs/custom_1_180_5be09c1dcce03.jpg',
+                'uploader': 'Sarah Calanthe',
+                'view_count': int,
+                'like_count': int,
+                'release_date': '20181110',
+                'tags': ['EyeContact', 'Interviews', 'MaskFetish', 'MouthFetish', 'Redhead'],
+                'release_timestamp': 1541851200,
+                'duration': 224.0,
+            },
         },
-    }]
+    ]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        real_url = f'https://www.manyvids.com/video/{video_id}/gtm.js'
-        try:
-            webpage = self._download_webpage(real_url, video_id)
-        except Exception:
-            # probably useless fallback
-            webpage = self._download_webpage(url, video_id)
+        info = traverse_obj(
+            self._download_json(f'https://www.manyvids.com/bff/store/video/{video_id}', video_id),
+            ('data', {dict})) or {}
 
-        info = self._search_regex(
-            r'''(<div\b[^>]*\bid\s*=\s*(['"])pageMetaDetails\2[^>]*>)''',
-            webpage, 'meta details', default='')
-        info = extract_attributes(info)
-
-        player = self._search_regex(
-            r'''(<div\b[^>]*\bid\s*=\s*(['"])rmpPlayerStream\2[^>]*>)''',
-            webpage, 'player details', default='')
-        player = extract_attributes(player)
+        video_urls = traverse_obj(
+            self._download_json(f'https://www.manyvids.com/bff/store/video/{video_id}/private', video_id),
+            ('data', {dict})) or {}
```

Comment:
don't need the traversal here. we have no video if we don't get `'data'`, so it can be fatal
```suggestion
        video_urls = self._download_json(
            f'https://www.manyvids.com/bff/store/video/{video_id}/private', video_id)['data']
```